cmd/proxsave/main_state.go

```
@@ -0,0 +1,85 @@
+}
+
+func newAppRunState() *appRunState {
+	return &appRunState{finalExitCode: types.ExitSuccess.Int()}
+}
+
+func (state *appRunState) finalize(code int) int {
+	state.finalExitCode = code
+	return code
+}
+
+func (state *appRunState) setBackupResult(result backupModeResult) {
```

golangci-lint (2.12.1)

[error] func (*appRunState).setBackupResult is unused

(unused)